### DISCUSSION TITLE
人人素材 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#37] [模板讨论]
人人素材 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
Failed at 12/17 request,
Fail assert: {"re": "错误", "from": "content"} from failed_asserts,
Request URL: https://www.rrcg.cn/plugin.php?id=jzsjiale_sms%3Ahomesms&act=phonemimalogin&loginsubmit=yes&inajax=1

**Reply by wjf0214** [reply]
带验证码登录的，OCR识别不是百分百准确，这种一般是验证码识别错误，登录失败了。
多试几次即可。

**Comment by miaoxiaozhao** [comment]
2025-09-24 14:53:53
日志：Failed at 1/5 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.rrcg.cn/plugin.php?id=k_misign%3Asign
